README.md

```diff
@@ -147,8 +147,40 @@ sudo dpkg -i postgresql-9.3-tsearch-extras_0.1.2_amd64.deb
 sudo apt-get install postgresql-9.4
 wget https://dl.dropboxusercontent.com/u/283158365/zuliposs/postgresql-9.4-tsearch-extras_0.1_amd64.deb
 sudo dpkg -i postgresql-9.4-tsearch-extras_0.1_amd64.deb
+```
+
+On Fedora 22:
```

Comment:
Maybe s/Fedora 22/Fedora 22 (experimental)/

And add a little bit of a heading mentioning that this is new and we'd appreciate pull requests, bug reports, and reports on whether the same process works fine for other Fedora versions?

I can also send mail to zulip-announce@ about this so we get some people trying it.